### PR TITLE
Update Dockerfile to ubi based dockerfile

### PR DESCRIPTION
**Pull request by waveywaves**
- Update dockerfile to ubi based dockerfile

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -50,4 +42,4 @@ RUN mkdir -p /home/builder
 ENV DOCKER_TLS_VERIFY   1
 ENV DOCKER_CERT_PATH    /minikube/certs
 
-ENTRYPOINT ["./entrypoint.sh"]
+ENTRYPOINT bash
```
Comment (akram) [inline Dockerfile:45]: is this temporary?

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
-FROM docker:18.09
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 
-ARG GO_VERSION
+ARG GO_VERSION=1.12.13
 ARG OPERATOR_SDK_VERSION
```
Comment (akram) [inline Dockerfile]: we should also probably fix a default OPERATOR_SDK_VERSION

File: Dockerfile [hunk 3]
```diff
@@ -50,4 +42,4 @@ RUN mkdir -p /home/builder
 ENV DOCKER_TLS_VERIFY   1
 ENV DOCKER_CERT_PATH    /minikube/certs
```
Comment (akram) [inline Dockerfile:43]: What is that for?

**Review by akram — COMMENTED** [review]
Otherwise, it looks good to me.
Can you please make the changes ?

File: Dockerfile (hunk 3 above)
Comment (waveywaves) [reply Dockerfile:43]: This is something used upstream to use minikube context.

File: Dockerfile (hunk 1 above)
Comment (waveywaves) [reply Dockerfile:45]: Yeah, till we fix the CI. But we need to find a better point of entry as even in the entrypoint.sh script it resolves to bash

File: Dockerfile (hunk 3 above)
Comment (waveywaves) [reply Dockerfile:43]: Let's keep it as it is for now

**Pull request merged**